build-helpers/notarize.js

```diff
@@ -12,7 +12,7 @@ exports.default = async function notarizing(context) {
   await notarize({
     appBundleId: 'org.ferdium.ferdium-app',
     appPath: `${appOutDir}/${appName}.app`,
-    ascProvider: 'B6J9X9DWFL',
+    ascProvider: '55E9FPJ93P',
```

Comment:
I'm imagining that this doesn't required to be a secret right?

Reply:
no - this is good.

Reply:
It appears in the certificate anyway if you look at it closely.